[15.0][IMP] `purchase_tag`: add parent tags

SilvioC2C opened the pull request.

SilvioC2C committed:
[IMP] purchase_tag: add parent tags

ivantodorovich reviewed (approved):
I'm not against the changes around the `color` field, though.. are they intended?

SilvioC2C commented:
@ivantodorovich Yep, they are:
1. I changed the `default` attribute in the field declaration to make it possible to override the default method if necessary
2. regarding the changes to the field declaration in the form view, I added the widget to display the color itself instead of the `Integer` value

gurneyalex commented:
/ocabot merge minor
